recipe/meta.yaml

```diff
@@ -20,6 +20,8 @@ outputs:
         strong:
           - cudatoolkit {{ cuda_compiler_version }}|{{ cuda_compiler_version }}.*
     requirements:
+      build:
+        - sysroot_linux-64 2.17  # [cuda_compiler_version == "11.0"]
```

Comment:
@beckermr, does it make sense to have this as a `run` dependency or maybe `run_exports`? Am wondering how we can ensure packages downstream from `nvcc` use GLIBC 2.17 if they are using CUDA 11.0.

Reply:
Yes, add this to `run`. You don't need it in `build`

Reply:
Wouldn't we want it in both to make sure that conda-build flags this as requiring GLIBC 2.17?

Reply:
why would you need the sysroot package to build the `nvcc` package?

Reply:
We [use the C compiler]( https://github.com/conda-forge/nvcc-feedstock/blob/f2ee3a4798aaaa95680f55113026dd7a06f69478/recipe/meta.yaml#L24-L25 ) to configure some things during the build. Maybe this should be in `host` and `run` instead of `build`?

Reply:
This package itself provides a compiler, right?

If so then it should have an export on `__glibc >=2.17` as @isuruf added below. 

If it is a compiler itself, then when in use it def needs the sysroot in run. 

The others are more subtle. It does not hurt to have it in build, so if you are worried you might add it.

Reply:
Sure no objections to the suggestions thus far. Just trying to make sure we have all of our bases covered 🙂

Part of the concern with `build` and `host` is we are [installing stuff in the compiler's sysroot]( https://github.com/conda-forge/nvcc-feedstock/blob/4712ce14ce41b7ce7d9f20d092c98392bd70eb6d/recipe/install_nvcc.sh#L77 ). So want to make sure we handle that correctly.

Reply:
you probably want the sysroot package in host then? you may want to ignore some of the run exports depending what the nvcc compiler needs in terms of glibc.

Reply:
Ok I think I've addressed this in the new commits. Please let me know what you think.